frontend/app/controllers/Joiner.scala

```diff
@@ -72,8 +73,13 @@ object Joiner extends Controller with ActivityTracking
       customSignInUrl=Some(signInUrl)
     )
 
-    Ok(views.html.joiner.tierChooser(TouchpointBackend.Normal.catalog, pageInfo, eventOpt, accessOpt, signInUrl))
-      .withSession(request.session.copy(data = request.session.data ++ contentRefererOpt.map(JoinReferrer -> _)))
+    contentRefererOpt.filter(Uri.parse(_).host.exists(_ == "www.theguardian.com")).map { contentURL =>
```

Comment:
I might be reading this wrong, but does this mean that if there was a link on theguardian.com to the tier chooser page and it was clicked by the user, they would see the restricted content message?